src/WordTokenizers.jl

```diff
@@ -4,7 +4,12 @@ module WordTokenizers
 using HTML_Entities
 using StrTables
 using Unicode
+using GoogleDrive
+using DataDeps
 
+abstract type Pretrained_tokenizer end
```

Comment:
I like having camel case for types. So `PretrainedTokenizer`?

Reply:
Okay I will do